docs/install/kubernetes.md

```diff
@@ -35,7 +35,21 @@ curl -sSLo kubectl-storageos.tar.gz \
 > 💡 You can find binaries for different architectures and systems in [kubectl
 > plugin](https://github.com/storageos/kubectl-storageos/releases).
 
-### Install Ondat
+### Option A: Install Ondat (embedded etcd)
```

Comment:
Embedded is a mode we used to have in v1. But it is not the same thing as we are expecting as pods. There is docs about it and some customers could get confused. It is unfortunate, but I think we should find an alternative to the word embedded. May be etcd as pods, etcd operator, etcd in k8s, or something like that.